backend/app/tests/api/routes/scheduled_inquiry/test_get_scheduled_inquiries_api.py

```diff
@@ -0,0 +1,48 @@
+from fastapi.testclient import TestClient
+from sqlmodel import Session
+
+from app.core.config import settings
+from app.models import InquiryCreate
+from app.services import inquiries as inquiries_service
+from app.services import scheduled_inquiries as scheduled_inquiries_service
+
+path = f"{settings.API_V1_STR}/scheduledinquiries/"
+
+
+def test_get_scheduled_inquiries_when_scheduled_inquiries_exist_should_return_all(
+    client: TestClient,
+    superuser_token_headers: dict[str, str],
+    db: Session,
+) -> None:
+    # Setup
+    inquiry_in_1 = InquiryCreate(text="Test inquiry 1.")
+    inquiry_in_2 = InquiryCreate(text="Test inquiry 2.")
+    db_inquiry_1 = inquiries_service.create_inquiry(session=db, inquiry_in=inquiry_in_1)
```

Comment:
Should we avoid calling other services when testing a service? I had a similar doubt when I was writing test for `get_inquiries` service, and I ended up using fixture that directly creates an inquiry in the table.

Reply:
Oh this is API test, but similar concern of isolation of unit test.

Reply:
That makes sense. I'll fix this